finagle-core/src/main/scala/com/twitter/finagle/client/BackupRequestFilter.scala

```diff
@@ -127,7 +117,23 @@ object BackupRequestFilter {
    *                       is returned and the result of the outstanding request is superseded. For
    *                       protocols without a control plane, where the connection is cut on
    *                       interrupts, this should be "false" to avoid connection churn.
+   *
+   * @param minSendBackupAfterMs Use a minimum non-zero delay to prevent sending unnecessary backup requests
+   *                             immediately for services where the latency at the percentile where a
+   *                             backup will be sent is ~0ms.
    */
+  def Configured(maxExtraLoad: Double, sendInterrupts: Boolean, minSendBackupAfterMs: Int): Param = {
+    require(
+      maxExtraLoad >= 0 && maxExtraLoad < 1.0,
+      s"maxExtraLoad must be between 0.0 and 1.0, was $maxExtraLoad"
+    )
+    require(
+      minSendBackupAfterMs >= 1,
+      s"minSendBackupAfterMs must be greater or equal to 1ms, was $minSendBackupAfterMs"
+    )
+    Param.Configured(Tunable.const(role.name, maxExtraLoad), sendInterrupts, minSendBackupAfterMs)
+  }
+
   def Configured(maxExtraLoad: Double, sendInterrupts: Boolean): Param = {
```

Comment:
hmmm, it looks like there are some duplicate code block, wdyt having one method call the other one? Maybe something like: 
`def Configured(maxExtraLoad: Double, sendInterrupts: Boolean): Param = Configured(maxExtraLoad, sendInterrupts, 1)`